policyengine_canada/variables/gov/provinces/bc/tax/income/credits/bc_disability_credit/bc_disability_credit_eligible.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_canada.model_api import *
+
+
+class bc_disability_credit_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for the British Columbia disability tax credit"
+    unit = CAD
+    definition_period = YEAR
+    defined_for = ProvinceCode.BC
+
+    def formula(person, period):
+        return person("is_disabled", period)
```

Comment:
Adding the age-eligible (<18) test to this variable can match the original policy better.